app/livechat/server/lib/RoutingManager.js

```diff
@@ -159,7 +159,13 @@ export const RoutingManager = {
 			return forwardRoomToDepartment(room, guest, transferData);
 		}
 
-		return false;
+		// no target agent or department specified, transfer to any online agent
+		const agent = await Users.getNextAgent();
```

Comment:
```suggestion
		const agent = await this.getNextAgent();
```

Reply:
You can't bypass the routing system to perform this action.
Please, remove the comments, we know what's supposed to be done here.

Reply:
Hello @renatobecker 
I've integrated the routing system as u suggested. I'd really appreciate it if u could review this PR again.